# TB-metabolomics-and-gene-expression-bioinformatics-analysis
Tuberculosis (TB) is a critical global public health issue, with a complex pathogenesis involving various metabolic and immune regulatory pathways. This study explores differences in metabolite levels and gene expression between TB patients and healthy controls using metabolomics and WGCNA (Weighted Gene Co-expression Network Analysis).
We recruited 74 patients with pulmonary tuberculosis (TB) and 106 healthy individuals undergoing physical examinations as a control group (HC) . Whole blood samples were collected and subjected to liquid chromatography-mass spectrometry (LC-MS) analysis. The raw data obtained were combined with datasets GSE19491 and GSE107994 retrieved from the Gene Expression Omnibus (GEO) for data analysis. OPLS-DA (Orthogonal Partial Least Squares Discriminant Analysis) was applied to identify metabolomic differences between TB and HC. GSEA (Gene Set Enrichment Analysis) and GSVA (Gene Set Variation Analysis) were used to evaluate key metabolites. A metabolite-gene-pathway network was constructed, and immune scoring was performed to analyze the relationships between key genes and immune cells.
